src/components/modals/createTaskModal.tsx

```diff
@@ -0,0 +1,238 @@
+import React, { useState, useEffect } from "react";
+import task_modal_style from "./createTaskModal.module.sass";
+import axios from "axios";
+
+function CreateTaskModal(props: any) {
+  const [contributors, setContributors] = useState([]);
+  const [ids, setId] = useState<any[]>([]);
+
+  function getContributors() {
+    let api = `http://127.0.0.1:8000/project/contributors/${props.id}/accepted/`;
+    axios.defaults.xsrfHeaderName = "X-CSRFTOKEN";
+    axios.defaults.xsrfCookieName = "csrftoken";
+    const token = window.localStorage.getItem("token");
+    axios.defaults.headers = {
+      "Content-Type": "application/json",
+      Authorization: `Token ${token}`,
+    };
+    axios.get(api).then((response) => {
+      setContributors(response.data);
+      console.log(response.data);
+    });
+  }
+
+  const [taskObjects, setTaskObjects] = useState({
+    name: "",
+    description: "",
+    start_date: "",
+    due_date: "",
+    assignees: [] as any,
+  });
+  const [times, setTime] = useState({
+    due_time: "",
+    start_time: "",
+  });
+  const handleTimeChange = (e: React.ChangeEvent<any>) => {
+    setTime({
+      ...times,
+      [e.target.name]: e.target.value,
+    });
+    console.log(times);
+  };
+
+  const handleChange = (e: React.ChangeEvent<any>) => {
+    setTaskObjects({
+      ...taskObjects,
+      [e.target.name]: e.target.value,
+    });
+    console.log(taskObjects);
+  };
+
+  const handleSubmit = (e: React.FormEvent) => {
+    e.preventDefault();
+    let api = `http://127.0.0.1:8000/project/create-task-list/${props.id}/`;
+    axios.defaults.xsrfHeaderName = "X-CSRFTOKEN";
+    axios.defaults.xsrfCookieName = "csrftoken";
+    const token = window.localStorage.getItem("token");
+    axios.defaults.headers = {
+      "Content-Type": "application/json",
+      Authorization: `Token ${token}`,
+    };
+
+    taskObjects.start_date = `${taskObjects.start_date} ${times.start_time}`;
```

Comment:
We have to merge the start time and start date in order for the django backend to accept it.